bundle/apps/upload_config.go

```diff
@@ -23,10 +22,9 @@ type uploadConfig struct {
 }
 
 func (u *uploadConfig) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
-	var diags diag.Diagnostics
+	var diags diag.ThreadsafeDiagnostics
```

Comment:
Nit: I believe a more ecosystem-compatible prefix is "Safe" or "Concurrent".

Reply:
We can go with Safe, it's shorter. 25838154cd2548f50959b9108833e2281c41bfdb